plugins/jira-dashboard/src/components/JiraTable/JiraTable.tsx

```diff
@@ -20,31 +26,47 @@ export const JiraTable = ({ tableContent }: Props) => {
   }
   const nbrOfIssues = tableContent?.issues?.length ?? 0;
 
+  const defaultFilters: TableFilter[] | undefined = [
+    {
+      column: 'Status',
+      type: 'multiple-select',
+    },
+    {
+      column: 'Priority',
+      type: 'multiple-select',
+    },
+  ];
+
   return (
-    <Table
+    <InfoCard
```

Comment:
I see what you are trying to do here, since the Table title is not working correctly when the filters are used. I see that in upstream Backstage's storybook, the [Filter Table](https://backstage.io/storybook/?path=/story/data-display-table--filter-table) doesn't use a title. Which is too bad for us.

However, can we explore different solutions to this? This to avoid having to add a InfoCard around the Table?

Reply:
We could possibly pull the title out of the table without the infocard.
Filters Closed
![image](https://github.com/user-attachments/assets/335e5a37-66e1-42af-b107-abb3cc25fc83)

Filters Open:
![image](https://github.com/user-attachments/assets/43be4db3-841b-4601-b29e-edac71762d8f)

Reply:
yes, that's what I was suggesting. Make it optional.

Reply:
> We could possibly pull the title out of the table without the infocard. Filters Closed ![image](https://private-user-images.githubusercontent.com/149509666/364376991-335e5a37-66e1-42af-b107-abb3cc25fc83.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1MjA0NTAsIm5iZiI6MTcyNTUyMDE1MCwicGF0aCI6Ii8xNDk1MDk2NjYvMzY0Mzc2OTkxLTMzNWU1YTM3LTY2ZTEtNDJhZi1iMTA3LWFiYjNjYzI1ZmM4My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwNVQwNzA5MTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wNjExZjVhZTdkNTAwMDUzZmUxYTQ5Y2MyOGZiNGQ2NzgxY2YwNGJlMWI2ZDg1YWQzM2Q3YjYzZGUxNjVlZmJjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ZE4klGAU6Lz8zq70aAkeE_gIEUILnYwWepT1sLGQ39E)
> 
> Filters Open: ![image](https://private-user-images.githubusercontent.com/149509666/364376779-43be4db3-841b-4601-b29e-edac71762d8f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1MjA0NTAsIm5iZiI6MTcyNTUyMDE1MCwicGF0aCI6Ii8xNDk1MDk2NjYvMzY0Mzc2Nzc5LTQzYmU0ZGIzLTg0MWItNDYwMS1iMjllLWVkYWM3MTc2MmQ4Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwNVQwNzA5MTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00YWFiODZlZGM3Njc1NTIwYmFmYzBlYzMyMDhjNzlhMTFiZjZiNmFhZDNjZWZhZjFlMDhmZWI0MzU4ZjI4NjdlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.FUOOgc3nSdcug_ByDbPaC5-OQVpKxeIRUibz9swKB0M)

Or what about keeping the InfoCard but removing it's box-shadow with boxShadow: 'none' ?
![Screenshot from 2024-09-05 09-20-17](https://github.com/user-attachments/assets/a145a4d9-28b4-492a-826e-a222450c1125)

Reply:
This works, however the noPadding argument provided to the info card in your screenshot results in the filters not displaying correctly.
Increasing the padding on the table style does not fix the padding on the filter component either.
![image](https://github.com/user-attachments/assets/188c4186-23a3-4daf-bfe6-2d1a7a6a7349)

Removing noPadding argument:
![image](https://github.com/user-attachments/assets/14ad3004-bb46-4a9e-b058-be7003e2b8ce)